src/popup/Popup.tsx

```diff
@@ -175,15 +177,27 @@ const DownloadCSVPanel: React.FC<{setcompanyNames: (companyNames: string[]) => v
                 Storing this list in local storage will enhance the extension`&apos;`s
                 performance and reduce unnecessary data usage.
             </Text>
+            <Text mt="sm" c="dimmed" size="sm">
+                You can get the csv URL from{' '}
+                <a href="https://www.gov.uk/government/publications/register-of-licensed-sponsors-workers" target="_blank" rel="noopener noreferrer"
+                style={{ color: 'var(--mantine-color-anchor)' }}> gov.uk website</a>.
+                <br/>
+                Please go to this page and copy the URL of the csv link.
+                <br/>
+                *The URL must end with
+                <br/>
+                20XX-MM-DD_-_Worker_and_Temporary_Worker.csv
+            </Text>
             <Input
                 placeholder="Download URL for the csv"
                 value={url}
                 onChange={e => setUrl(e.target.value)}
                 mt="md"
             />
             <div className="reverse-flex">
-                <Button loading={isLoading} size="xs" variant="filled" onClick={handleDownloadClicked}>Save CSV</Button>
+                <Button m={10} loading={isLoading} size="xs" variant="filled" onClick={handleDownloadClicked}>Save CSV</Button>
             </div>
         </>
```

Comment:
Now that there is a container, we can remove this react fragment.

React enforces that every compondnt returns a single "node", and so fragments are commonly used as wrapper nodes in components where we want to return multiple elements.

They have no effect on the generated HTML. They only exist in the virtual DOM.

Reply:
I see, I removed that fragment.